source/main_cr.f90

```diff
@@ -731,20 +731,16 @@ program maincr
 
   ! save tas matrix and closed orbit for later dumping of the beam
   ! distribution at the first element (i=-1)
-  ! dumptas(*,*) [mm,mrad,mm,mrad,1] canonical variables
```

Comment:
Keep this information where `type, public :: dump_normType` is defined?

Reply:
I.e. just the units etc. -- the "matrix is different for each particle" is outdated now, right?

Reply:
No idea what you're referring to here. Please explain.

Reply:
I meant to put the description about the units (corrected if necessary) into where the dump_normType is described, as there is no precise description of the variables there now.

Reply:
Cleaned up these comments anyway, and added a note to check conversion to canonicals later. Something doesn't look quite right.